### DISCUSSION TITLE
TimeInput in Firefox

### DISCUSSION THREAD

**Discussion by lokeshwarobuli** [mantinedev/mantine#4606] [Q&A]
How to enable the timeinput picker in firefox. Even the sample playground is not working when I open https://mantine.dev/dates/time-input/#show-browser-picker in firefox and try to click the icon

Thanks in advance

**Comment by rtivital** [comment]
Firefox does not support `showPicker` method on time input – https://caniuse.com/mdn-api_htmlinputelement_showpicker_time_input

**Reply by lokeshwarobuli** [reply]
Is there any workaround, or in general it is not possible ?

**Reply by rtivital** [reply]
It is not supported by Firefox as displayed in the table. You can create your own UI for this with Popover component if you need this feature.
